tui-popup/src/popup.rs

```diff
@@ -144,7 +120,7 @@ impl<W: KnownSize + WidgetRef> StatefulWidget for &Popup<'_, W> {
         let inner_area = block.inner(popup_area);
         block.render(popup_area, buf);
 
-        self.body.render_ref(inner_area, buf);
+        self.body.clone().render(inner_area, buf);
```

Comment:
This one line is the crux of this entire change. I can't get this to work with &Widget, the best I can do is Clone the inner body widget. I'm not sure if this would be better left using WidgetRef instead of requiring `Clone`